website/static/js/registrationUtils.js

```diff
@@ -153,6 +185,17 @@ var Question = function(data, id) {
             return new Comment(comment);
         })
     );
+	self.deleteComment = function(comment) {
+		self.comments.remove(comment);
+		comment.isDeleted = true;
+
+		var commentList = document.getElementById('commentList');
```

Comment:
No need to modify any DOM elements. Let KO do this for us. Keep deleted comments in the order they were originally created, just clear their text and make them non-editable.